docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -1,763 +1,166 @@
 ---
 title: Run E2E tests on EAS Build
 sidebar_title: Run E2E tests
-description: Learn how to set up and run E2E tests on EAS Build with popular libraries such as Detox.
+description: Learn how to set up and run E2E tests on EAS Build with Maestro.
 ---
 
-import { GithubIcon } from '@expo/styleguide-icons/custom/GithubIcon';
-
-import { BoxLink } from '~/ui/components/BoxLink';
-import { Collapsible } from '~/ui/components/Collapsible';
 import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
-import { Terminal, DiffBlock } from '~/ui/components/Snippet';
+import { Terminal } from '~/ui/components/Snippet';
 import { Step } from '~/ui/components/Step';
 
-> **Warning** EAS Build support for E2E testing is in a _very early_ state. This guide intends to explain how you can run E2E tests on the service today, without all of the affordances that we plan to build in the future. This guide will evolve over time as support for testing workflows in EAS Build improves.
-
-With EAS Build, you can build a workflow for running E2E tests for your application. In this guide, you will learn how to use one of the most popular libraries ([Detox](https://wix.github.io/Detox)) to do that.
-
-This guide explains how to run E2E tests with Detox in a bare workflow project. You can use [`@config-plugins/detox`](https://github.com/expo/config-plugins/tree/main/packages/detox) for a managed project, but you may need to adjust some of the instructions in this guide to do so.
+> **Warning** This guide will evolve over time as support for testing workflows in EAS Build improves.
 
-## Running iOS tests
+In this guide, you will learn how to run E2E tests on EAS Build using one of the most popular tools for running E2E tests in mobile apps - [Maestro](https://maestro.mobile.dev/).
+In the following steps, we will try to show you using an example how you can configure your EAS Build Maestro E2E tests workflow today:
```

Comment:
```suggestion
In this guide, you will learn how to create and run E2E tests on EAS Build using [Maestro](https://maestro.mobile.dev/), which is one of the most popular tools for running E2E tests in mobile apps.

In the following steps, the example demonstrates how to configure your EAS Build Maestro E2E tests workflow using the [default Expo template](/more/create-expo/#--template). For your own app, you will need to adjust the flows to match your app's UI.
```

Let's divide this into two paragraphs and also callout that this guide demonstrates an example using the default template. adding that callout in step 3 is somewhat surprising from end user's perspective. We can also link to the default template reference.